src/annotator/test/integration/multi-frame-test.js

```diff
@@ -24,43 +23,40 @@ describe('CrossFrame multi-frame scenario', () => {
       call: sandbox.stub(),
       destroy: sandbox.stub(),
     };
-    fakeAnnotationSync = { destroy: sandbox.stub() };
-    proxyAnnotationSync = sandbox.stub().returns(fakeAnnotationSync);
-    proxyBridge = sandbox.stub().returns(fakeBridge);
-
-    $imports.$mock({
-      './annotation-sync': { AnnotationSync: proxyAnnotationSync },
-      '../shared/bridge': proxyBridge,
-    });
+    hypothesisInjectors = [];
 
     container = document.createElement('div');
     document.body.appendChild(container);
-
-    config = {
-      clientUrl: 'data:,', // empty data uri
-    };
-
-    crossFrame = null;
   });
 
   afterEach(() => {
     sandbox.restore();
-    crossFrame.destroy();
-    container.parentNode.removeChild(container);
-
-    $imports.$restore();
+    hypothesisInjectors.forEach(hypothesisInjector =>
+      hypothesisInjector.destroy()
+    );
+    container.remove();
```

Comment:
I believe this should do the same as the previous line:
`container.parentNode.removeChild(container);`

Reply:
You're right. The use of the `element.parentNode.removeChild(element)` pattern in a few places is a legacy from when the test environment didn't support the `Element.remove` method.